symphony-bdk-examples/bdk-core-examples/src/main/java/com/symphony/bdk/examples/ServiceExampleMain.java

```diff
@@ -37,8 +38,15 @@ public static void main(String[] args) throws Exception {
     log.info("Role: {}", userDetailList.get(0).getRoles());
 
     AuthSession oboSession = bdk.obo("hong.le");
+
     StreamFilter streamFilter = new StreamFilter().addStreamTypesItem(new StreamType().type(StreamType.TypeEnum.IM));
-    List<StreamAttributes> streamsList = bdk.obo(oboSession).streams().listStreams(streamFilter);
+    final List<StreamAttributes> streamsList = bdk.obo(oboSession).streams().listStreams(streamFilter);
     log.info("Streams List: {}", streamsList);
+
+    final List<Signal> signalList = bdk.obo(oboSession).signals().listSignals();
+    log.info("Retrieve {} signals", signalList.size());
+    log.info("First signal:");
+    log.info("Signal name: " + signalList.get(0).getName());
```

Comment:
we are mixing {} and string concat (not really a performance problem here but just to be consistent)